openedx/adg/common/lib/mandrill_client/tasks.py

```diff
@@ -55,3 +55,25 @@ def task_cancel_mandrill_emails(msg_ids):
     for msg_id in msg_ids:
         log.info(f'Cancelling a scheduled email, mandrill msg_id: {msg_id}')
         MandrillClient().cancel_scheduled_email(msg_id)
+
+
+@task()
+def task_send_mandrill_email_for_email_address_change(template, context, old_email, new_email):
+    """
+    Sends email address change email by calling send_mandrill_email, asynchronously.
+
+    Reason to create new task is that, we want to send email in the language that user has selected. In case
+    of email address changes we do not have preference for new email address so we are using the old email address.
```

Comment:
Let's rephrase this.
In case of email address change, we need to get the user language preference from the old email address before sending the correct email template to the user.